index.html

```diff
@@ -83,37 +84,181 @@ <h5 class="card__title"><img src="./img/svg/suitcase.svg" class="card-icon"
                         Marketing</h5>
-                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog
+                    </p>
                 </div>
 
                 <div class="grid-card">
                     <h5 class="card__title"><img src="./img/svg/notebook.svg" class="card-icon"
                             alt="notebook image">Business Growing</h5>
-                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog
+                    </p>
                 </div>
 
                 <div class="grid-card">
                     <h5 class="card__title"><img src="./img/svg/clients.svg" class="card-icon"
                             alt="clients image">National top 50 firms</h5>
-                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog
+                    </p>
                 </div>
 
                 <div class="grid-card">
                     <h5 class="card__title"><img src="./img/svg/branches.svg" class="card-icon"
                             alt="branches image">National top 50 firms</h5>
-                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog </p>
+                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog
+                    </p>
                 </div>
 
                 <div class="grid-card">
                     <h5 class="card__title"><img src="./img/svg/award.svg" class="card-icon" alt="award image">Web
                         Development</h5>
-                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+                    <p class="card__text">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog
+                    </p>
                 </div>
+            </div>
+        </div>
+    </section>
 
+    <section class="customer-feedback">
+        <div class="container">
+            <div class="feedback__content">
+                <h2 class="content__general-title">What Clients Say</h2>
+                <p class="content__text">Problems trying to resolve the conflict between <br> the two major realms of
+                    Classical physics: Newtonian mechanics<p>
             </div>
 
+            <div class="grid-container">
+                <div class="grid-card" id="card-items">
+                    <div class="stars">
+                        <span data-rate="1"></span>
```

Comment:
I think you could use simple images here and add only width and height in css, so in this way you could have saved some code. But i guess it works both ways :)

Reply:
I`ve added url with images (svg) to .stars and .empty-stars classes. I found this way to display the rating and I also have a question if it`s ok to use this way and to bother with it))

Thank you a lot for your review!